source/nodejs/adaptivecards-designer/src/designer-peers.ts

```diff
@@ -1685,7 +1689,11 @@ export class ActionSetPeer extends TypedCardElementPeer<Adaptive.AdaptiveCard> {
     }
 }
 
-function createActionSelector(title: string, card: Adaptive.AdaptiveCard, defaultValue: string): ILabelAndInput<Adaptive.ChoiceSetInput> {
+function createActionSelector(
```

Comment:
Nit: Would you mind creating tracking issues for us to start adding xml-syle function/module level comments where needed? We've started tracking adding these to other source files as well. Thanks!

Reply:
Opened https://github.com/Microsoft/AdaptiveCards/issues/2789